_config.typography.scss

```diff
@@ -0,0 +1,167 @@
+
+// Base font stack
+$font: Arial, Helvetica, sans-serif;
+$font-secondary: "Arial Black", sans-serif;
+
+// Base font weights
+$regular: 400;
+$bold: 700;
+
+// Initialise font use on the html element
+html {
+    font-family: $font;
+}
+
+// Arial Black (extra bold)
+// The brand guidelines specify an additional third font weight option, Arial Black. Arial Black is a completely separate font file to Arial (defined above as $font) so that needs to be dealt with differently. To use Arial Black, either put the following class directly into your HTML. Alternatively, you can use Sass @extend (ref: http://sass-lang.com/documentation/file.SASS_REFERENCE.html#_extend__extend) to pull the contents of the class into another CSS rule or just use the $font-secondary variable.
+
+.pr-u-arial-black {
```

Comment:
Not an issue, but am curious as to reasons for class name being font-specific rather than a generic "secondary-font" class or similar?